### PR TITLE
Removes South Bronx Paradise Bar

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
as title

## Why It's Good For The Game
As funny as it is I think it's kinda mean to have a trap item just. Right there

## Changelog

:cl:
del: South Bronx Paradise Bar
/:cl:

### CODE REVIEW

**Comment by tantspants** [comment]
thank you, that thing was fucking annoying

**Comment by thgvr** [comment]
soulless

**Comment by Erikafox** [comment]
thg you know that im reaving that soul.

**Comment by kaylexis** [comment]
soul removal back at it again. i'd like some more parasites, waiter

**Comment by thgvr** [comment]
can you actually just take this out of vending machines and put it on sand planets or ruins or something

**Comment by Xkallubar** [comment]
Just move it to the contraband section of the vending machine like it is on every other server?

**Comment by Erikafox** [comment]
> can you actually just take this out of vending machines and put it on sand planets or ruins or something

fun fact i was going to do that and figured someone would tell me to just remove it

>Just move it to the contraband section of the vending machine like it is on every other server?

:clueless:

**Comment by MassiveMen** [comment]
another vending machine sterilized

**Comment by Martinpachu** [comment]
but i need parasites to live.

**Comment by thgvr** [comment]
> Just move it to the contraband section of the vending machine like it is on every other server?

what is blud yappin about

**Comment by Xkallubar** [comment]
> > Just move it to the contraband section of the vending machine like it is on every other server?
> 
> what is blud yappin about

Unless it's different on this server, you just need to get the green light to turn on when tinkering with vending machine wires to access hidden items. Is that not a thing here?

**Comment by Anticept** [comment]
It is, but it wont be. The entire contraband thing is going to be gone. That's why there's goofy replies about it.

**Comment by Sun-Soaked** [comment]
what if it just made you vomit instead of the worms (the worms)(the w

**Comment by meemofcourse** [comment]
worm funny though

**Comment by Erikafox** [comment]
please take all discussion to the discord. this pr is feature complete.

**Comment by Erikafox** [comment]
That is to say im not changing it.

File: code/modules/food_and_drinks/food/snacks_vend.dm [hunk 1]
```diff
@@ -15,36 +15,6 @@
...
-	tastes = list("candy" = 5, "weight loss" = 4, "insect larva" = 1)
-	foodtype = JUNKFOOD | RAW | GROSS
-	custom_premium_price = 800
-	var/revelation = FALSE
-
-/obj/item/reagent_containers/food/snacks/candy/bronx/On_Consume(mob/living/eater)
-	. = ..()
-	if(ishuman(eater))
-		var/mob/living/carbon/human/carl = eater
-		var/datum/disease/P = new /datum/disease/parasite()
```
Comment (SomeguyManperson) [inline code/modules/food_and_drinks/food/snacks_vend.dm:37]: Vaporize the disease too please it sucks
"Your liver is coming out one way or another" is stupif

**Comment by Erikafox** [comment]
will undraft when i do that tomorrow

**Comment by Erikafox** [comment]
done

**Comment by SomeguyManperson** [comment]
If anyone wants the worm bar back they are free to re-add it with a better punishment system. It works off virology so its trigger time is completely random and the only way to fix "your liver is coming out" is to remove your liver which is counterintuitive and not a solution for any ship without a medical bay

**Pull request merged**